allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -50,6 +50,7 @@ class PretrainedTransformerTokenizer(Tokenizer):
         - 'do_not_truncate': Do not truncate (raise an error if the input sequence is longer than max_length)
     calculate_character_offsets : `bool`, optional (default=False)
         Attempts to reconstruct character offsets for the instances of Token that this tokenizer produces.
+    tokenizer_kwargs: 'Dict[str, Any]' - dictionary with additional arguments for AutoTokenizer.from_pretrained.
```

Comment:
Does this format work with our docs?  Shouldn't the description be on the next line?

Reply:
Oh just copy pasted from that previous PR. Will update.